tap_xero/schemas/allocations.json

```diff
@@ -11,7 +11,7 @@
       ],
       "format": "date-time"
     },
-    "Amount": {
```

Comment:
I would be very careful / advise against removing the `Amount` key here. I don't see `Amount` mentioned in the Xero documentation, but that's true of much of Xero's API documentation. This key likely exists here from being auto-generated based on real data received from Xero.

Reply:
Agreed. I would add a new entry for `AppliedAmount`